test/features/organization_membership_test.rb

```diff
@@ -0,0 +1,40 @@
+require "test_helper"
+
+class EditOrganizationTest < Capybara::Rails::TestCase
+
+  def user
+    @user ||= users :normal
+    @user.password = 'password'
+    @user
+  end
+
+  def organization
+    @organization ||= organizations :mapc
+  end
+
+  alias_method :org, :organization
+
+  test "signed in user visits existing organization and requests to join it" do
+    sign_in user, visit: true, submit: true
+    visit organization_path(org)
+    click_link 'Request to Join'
+    assert_content page, 'Membership request sent'
+  end
+
+  test "signed in user visits existing organization, requests to join it twice, and receives an error the second time" do
+    sign_in user, visit: true, submit: true
+    visit organization_path(org)
+    click_link 'Request to Join'
+    click_link 'Request to Join'
+    assert_content page, 'Organization has already been taken'
```

Comment:
Hmm this is less straightforward: http://stackoverflow.com/questions/2680016/ruby-on-rails-how-to-get-error-messages-from-a-child-resource-displayed.